features/kiesrecht.feature

```diff
@@ -108,46 +108,235 @@ Rule: Uitsluiting van Europees kiesrecht wordt alleen opgenomen wanneer de eindd
-		| naam                                | waarde                                        |
-		| uitgeslotenVanKiesrecht             | <uitgeslotenVanKiesrecht>                     |
-		| einddatumUitsluitingKiesrecht.type  | <type>                                        |
-		| einddatumUitsluitingKiesrecht.datum | <einddatum>                                   |
-		| einddatumUitsluitingKiesrecht.jaar  | <einddatum jaar>                              |
-		| einddatumUitsluitingKiesrecht.maand | <einddatum maand>                             |
+  Rule: Uitsluiting van kiesrecht wordt opgenomen wanneer er geen einddatum uitsluiting van kiesrecht is
 
-		Voorbeelden:
-		| omschrijving                       | uitsluiting kiesrecht | einddatum uitsluiting | uitgeslotenVanKiesrecht | type           | einddatum  | einddatum jaar | einddatum maand |
-		| niet opgegeven                     |                       |                       |                         |                |            |                |                 |
-		| uitgesloten zonder einddatum       | true                  |                       | true                    |                |            |                |                 |
-		| einddatum uitsluiting in toekomst  | true                  | 20300101              | true                    | Datum          | 2030-01-01 |                |                 |
-		| einddatum uitsluiting in verleden  | true                  | 20220301              |                         |                |            |                |                 |
-		| einddatum uitsluiting vandaag      | true                  | 20220315              |                         |                |            |                |                 |
-		| einddatum uitsluiting morgen       | true                  | 20300316              | true                    | Datum          | 2030-03-16 |                |                 |
-		| einddatum uitsluiting vorige maand | true                  | 20220200              |                         |                |            |                |                 |
-		| einddatum uitsluiting deze maand   | true                  | 20220300              | true                    | JaarMaandDatum |            | 2022           | 3               |
-		| einddatum uitsluiting vorig jaar   | true                  | 20210000              |                         |                |            |                |                 |
-		| einddatum uitsluiting dit jaar     | true                  | 20220000              | true                    | JaarDatum      |            | 2022           |                 |
-		| einddatum uitsluiting onbekend     | true                  | 00000000              | true                    | DatumOnbekend  |            |                |                 |
+    Scenario: persoon is uitgesloten van kiesrecht en er is geen einddatum uitsluiting van kiesrecht
```

Comment:
aan de nieuwe scenario's is geen @proxy of @gba toegevoegd

Reply:
Klopt. Ik denk dat het niet nodig is om scenario's die nu voor de proxy gelden gemarkeerd moeten worden omdat die ook moeten gelden als de proxy ook is uitgefaseerd. Het is dan niet handig om alle features dan na te lopen om dit allemaal te verwijderen. Het is wel handig om de scenario's die specifiek zijn voor de GBA variant van de API omdat deze niet kunnen worden geautomatiseerd en daarom ook niet moeten worden uitgevoerd tijdens het valideren van de proxy

Reply:
Deze features moeten nu ook worden gebruikt voor het testen van de @gba implementatie. Hoe weten zij nu het onderscheid tussen een scenario die voor zowel @gba als @proxy geldt (zoals fields) of nu voor @proxy en na uitzetten van de proxy voor de API, maar dus niet voor @gba?